bin/fix_spaces.py

```diff
@@ -0,0 +1,22 @@
+import os
+import sys
+
+'''
+At least one tour (iwtomics) uses incorrect indentation that causes yamllint
```

Comment:
should we just yamllint the tours then? Surely would be better than needing an extra script

Reply:
Yes, this is script is not one of the "deliverables" and will likely be removed in the future.  Yamllint was complaining about spaces and I wanted to see if this made it happy (it does).  A better solution would be to fix the spacing in the original tutorial.